internal/scanner/scanner.go

```diff
@@ -2020,6 +2021,7 @@ func init() {
 	for text, key := range textToToken {
 		tokenToText[key] = text
 	}
+	maps.Copy(textToToken, textToKeyword)
```

Comment:
I'd consider putting this up by `textToToken`; packages can have more than one init function so you can just write it there. Or honestly, in these cases if I had written them originally I would have used an IIFE.